.github/workflows/regression.yml

```diff
@@ -9,14 +9,20 @@ on:
     branches: 
       - 'develop'
       - 'master'
+  workflow_call:
+    # workflows: ["Regression and Unit Post-Validation on Linux ARM64"]
+    # types: [requested]
+    # branches:
+      # - 'develop'
```

Comment:
The lines will be uncommented later to lock the caller only to `scheduled-arm64.yml`